### PR TITLE
7

### PR DESCRIPTION
**Pull request by partouf**
Improved wallet and menu mechanics, added items, menus, story components etc

### CODE REVIEW

File: Assets/scripts/game/SolarOSSocialMedia.cs [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                InitVenusPublic();
+            }
+        }
+
+        /// <summary>
+        /// Public shares on Venus
+        /// </summary>
+        private void InitVenusPublic()
+        {
+            publicMessages.Add(new SolarOSMenuItem(
```
Comment (partouf) [inline Assets/scripts/game/SolarOSSocialMedia.cs]: should probably make this read the messages from a resources file...

File: Assets/scripts/game/SolarOS.cs [hunk 2]
```diff
@@ -52,11 +52,18 @@ public class SolarOS : LevelBase
...
+            Venref = 5,
+
+            EarthMoon = 8,
+
+            Mars = 16,
+
+            Ceres = 24,
+
+            Europa = 32
         }
```
Comment (guytenaerefe) [inline Assets/scripts/game/SolarOS.cs:68]: Will there be combinations of networks?

File: Assets/scripts/items/KnownItemsInventory.cs [hunk 3]
```diff
@@ -6,9 +6,14 @@
     public class KnownItemsInventory : ItemInventory
     {
         /// <summary>
-        /// I'm here to kick ass and chew bubblegum, and I got 1 BubbleGumWrappingPaper.
```
Comment (guytenaerefe) [inline Assets/scripts/items/KnownItemsInventory.cs:9]: :dancing_men:

File: Assets/resources/news.venus.txt [hunk 4]
```diff
@@ -0,0 +1,14 @@
+﻿Venus News Network
+
+Local news
+* Mayor visits eldest Venutian
+
+Economics
+* Solarbits gaining on Dollar, Euro stays stable
```
Comment (guytenaerefe) [inline Assets/resources/news.venus.txt:7]: :man_dancing:

File: Assets/scripts/game/SolarOSSocialMedia.cs [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+                "@helawyr\n" +
+                "yet another day with more bad news from @voasis - o2 supply bottleneck caused by unknown malfunction"));
+            publicMessages.Add(new SolarOSMenuItem(
+                "@lexcant\n" +
+                "sometimes i think all i need is wine and catnip, and then it hits me, this is indeed all i need"));
+            publicMessages.Add(new SolarOSMenuItem(
+                "@officialvnn\n" +
+                "earth-moon sources say yearly progress report on earth climate recovery process will be as expected - no recovery within 10yrs. more at news.venus/climate"));
+            publicMessages.Add(new SolarOSMenuItem(
+                "@4thhand\n" +
```
Comment (guytenaerefe) [inline Assets/scripts/game/SolarOSSocialMedia.cs:164]: Nice Easter Egg

**Review by guytenaerefe — APPROVED** [review]
LGTM. Check comments

**Pull request merged**